targets/circuitplay-bluefruit.json

```diff
@@ -0,0 +1,9 @@
+{
+    "inherits": ["nrf52840"],
+    "build-tags": ["circuitplay_bluefruit"],
+    "flash-method": "msd",
+    "msd-volume-name": "CPLAYBTBOOT",
+    "msd-firmware-name": "firmware.uf2",
+    "uf2-family-id": 2913282112,
```

Comment:
See my previous comment about making this a string then calling `hex.DecodeString()` to make it more readable.

Reply:
Done